src/components/VSourcesTable.vue

```diff
@@ -90,10 +90,10 @@ export default defineComponent({
   setup(props) {
     const sorting = reactive({
       direction: 'asc',
-      field: 'display_name',
+      field: 'display_name' as keyof Omit<MediaProvider, 'logo_url'>,
```

Comment:
I wonder if we should remove the provider `logo_url` parameter. Do we use it anywhere?

Reply:
No, I don't think we are showing the sources logo anywhere, though it was the intention in CC to show them on the homepage if I recall correctly? we can probably remove them!